iOSDFULibrary/Classes/Utilities/Streams/DFUStreamZip.swift

```diff
@@ -123,7 +124,7 @@ internal class DFUStreamZip : DFUStream {
                 
                 // Look for and assign files specified in the manifest
                 let softdeviceBootloaderType = FIRMWARE_TYPE_SOFTDEVICE | FIRMWARE_TYPE_BOOTLOADER
-                if type & softdeviceBootloaderType == softdeviceBootloaderType {
+                if type.rawValue & softdeviceBootloaderType == softdeviceBootloaderType {
```

Comment:
Instead of using raw value can't we set `FIRMWARE_TYPE_SOFTDEVICE` and `FIRMWARE_TYPE_BOOTLOADER` to the equivalent enums of `DFUFirmwareType` ?
this is not a breaker but it would be a tiny bit cleaner

Reply:
FIRMWARE_TYPE_... are not visible in the DFUFirmwareType enum definition. I guess it's because the constants are internal and we don't want people to use them, but the enum instead. I would say "it's complicated".